HISTORY.rst

```diff
@@ -7,6 +7,7 @@ v0.26 (unreleased)
 ..................
 * fix to schema generation for ``IPvAnyAddress``, ``IPvAnyInterface``, ``IPvAnyNetwork`` #498 by @pilosus
 * fix variable length tuples support, #495 by @pilosus
+* fix ``.dict(skip_keys=True)`` skipping values set via alias, #517 by @sommd
```

Comment:
I think to be safe we need `**Breaking Change:**` etc here with a description of the change to `validate_model`, people shouldn't be using it directly, but I bet some are.